designs/tripleo/osc-tripleo.md

```diff
@@ -68,7 +68,7 @@ description: >
     description: image
     type: string
 	
-  # The .
+  # The source of the mounted volume OSC needs to persist information.
   OSCVolumeSrc:
     description: image
```

Comment:
unrelated but not sure i understand what the `description : image` is supposed to mean